docs/conf.py

```diff
@@ -0,0 +1,293 @@
+# -*- coding: utf-8 -*-
+#
+# envoy documentation build configuration file, created by
+# sphinx-quickstart on Sat May 28 10:51:27 2016.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import sphinx_rtd_theme
+import sys
+import os
+import git
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#sys.path.insert(0, os.path.abspath('.'))
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinxcontrib.httpdomain', 'sphinx.ext.extlinks']
+
+extlinks = {'repo': ('https://github.com/envoyproxy/envoy/blob/master/%s', '')}
+
+# Add any paths that contain templates here, relative to this directory.
+#templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The encoding of source files.
+#source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'envoy'
+copyright = u'2016-2017, Lyft'
```

Comment:
change?

Reply:
See above comment on `conf.py` being temporary, we will be nuking this as soon as we do cross linking with the main repo docs.

Reply:
@rshriram please just fix all of this in the main repo. I don't actually know of the copyright changes in this case or not. We will need to ask @caniszczyk.